content/monitors/monitor_uptime_widget.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Monitor Uptime Widget
+kind: documentation
+description: "Track your monitor’s uptime via a widget in your Screenboard"
+private: true
+disable_toc: true
+---
+
+<div class="alert alert-warning">
+The Monitor Uptime Widget is currently in private beta. Interested in trying it out?  <a href="https://docs.google.com/forms/d/1EITy8IbRe484X-vfVyX8foU-_QXMyYKB6ciKXh3YUvU/edit">Request private beta access</a>.
+</div>
+
+{{< img src="monitors/monitor_uptime_widget.png" alt="monitor uptime widget" responsive="true" >}}
+
+Use the monitor Uptime widget to track your monitor’s uptime via a widget in your [Screenboards][1]. Monitor Uptime is defined as the amount of time a monitor was in an up state (OK) compared to down state (non-OK). Similar to the monitor status page visualization, the status is represented in bars as green (up) and red (down).
+
+## Widget configuration
+
+{{< img src="monitors/monitor_uptime_widget_configuration.png" alt="monitor uptime widget configuration" responsive="true" >}}
+
+To configure the widget from a [Screenboard][1].
+
+1. Select the monitor to display uptime from.
+    Optionally - Select the monitor group ordering between the worst performing 5 groups, best performing 5 groups, or custom (select up to 5 custom groups). The total percentage when selected by group can be calculated: Based on the total uptime of the monitor or the average uptime for the selected groups.
```

Comment:
Can this portion start on a new line? 
Optionally, you can view uptime by group for that monitor in three different ways: 
Worst performing 5 groups
Best performing 5 groups
Custom (select up to 5 groups) 
The total percentage when selected by group can be calculated based on the total of the monitor, or the average for the selected groups.